lib/simplekiq/orchestration_job.rb

```diff
@@ -8,8 +8,10 @@ module OrchestrationJob
 
     extend Forwardable
     def_delegators :orchestration, :run, :in_parallel
+    attr_reader :orchestration
```

Comment:
I think this attr_reader is unused/unneeded after all the refactors

Reply:
Not quite, `build_orchestration` sets the ivar, and then the `attr_reader` gets used again in `workflow_plan`

Reply:
ah right... I keep forgetting about that method and tbh I kind of think we should just get rid of it... it's not used in any production code, it's really just there as a test helper, and it confuses the flow of this class (calling `perform_orchestration` but not `perform`) a lot

Reply:
But that is heading into a bigger refactor not relevant to this PR, ignore me for now